components/compliance-service/reporting/relaxting/util.go

```diff
@@ -291,3 +291,14 @@ func StringTagsFromProtoFields(tKey string, tValue *structpb.Value) *ESInSpecRep
 	}
 	return nil
 }
+
+func FetchLatestDataOrNot(filters map[string][]string) bool {
+	fmt.Println(filters)
```

Comment:
Is this print statement required? If yes, then can we replace it with a log message?